README.md

```diff
@@ -13,6 +13,55 @@ Openmrs-module-queue
 - Java 8 or higher
 - Webservices rest module -(Always bundled with the platform)
 
+## Docker development environment
+Prepare the docker development environment by executing the following command;
+
+```bash
+sh prepare-docker-env.sh
```

Comment:
So the additional step is mainly for downloading the other required modules for this module to run. I suppose we could add to the entry point script for the dev image. How can I make sure we are not redownloading the same modules every time we run the Image?

Reply:
I would make it so that deploy_module.sh simply checks if a directory with modules exists. If it does then it does not re-downloads. If one needs to apply changes then the script could accept a parameter e.g. '-r' or '--refresh'. It's simple and it does not force to remember to run a different script.